src/App.css

```diff
@@ -33,10 +33,40 @@
   }
 }
 
-.card {
+ .card {
   padding: 2em;
 }
 
 .read-the-docs {
   color: #888;
 }
+
+
+.cars-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 980px;
+}
+
+.left {
+  flex: 1;
+}
```

Comment:
There is no difference between `.left` and `.right` in your code right? so it can be a single class

Reply:
"left" is renamed to "car-form-add" and "right" is removed because it is not used